### PR TITLE
Oracle fix

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: internal/contracts/oracle_contract/go.mod [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module github.com/nspcc-dev/neo-go/examples/oracle
+
+go 1.16
```
Comment (AnnaShaleva) [inline internal/contracts/oracle_contract/go.mod:3]: 1.17

File: internal/contracts/oracle_contract/oracle.go [hunk 2]
```diff
@@ -22,3 +25,18 @@ func Handle(url string, data interface{}, code int, res []byte) {
...
+	// Regular safety check.
+	callingHash := runtime.GetCallingScriptHash()
+	if !callingHash.Equals(oracle.Hash) {
+		panic("not called from oracle contract")
+	}
+
+	runtime.Notify("Invocation")
+	if runtime.GetInvocationCounter() == 1 {
+		// We provide no wrapper for finish in interops, it's not usually needed.
+		contract.Call(interop.Hash160(oracle.Hash), "finish", contract.All)
```
Comment (AnnaShaleva) [inline internal/contracts/oracle_contract/oracle.go:40]: Hm.

File: internal/contracts/oracle_contract/go.mod (hunk 1 above)
Comment (roman-khimov) [reply internal/contracts/oracle_contract/go.mod:3]: This branch waited too long...

Comment (roman-khimov) [reply internal/contracts/oracle_contract/go.mod]: Fixed.

File: internal/contracts/oracle_contract/go.mod [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module github.com/nspcc-dev/neo-go/examples/oracle
+
+go 1.17
+
+require github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20220713145417-4f184498bc42
```
Comment (AnnaShaleva) [inline internal/contracts/oracle_contract/go.mod:5]: And probably dependency checker is complaining due to this line. We need to update the interop version.

Comment (roman-khimov) [reply internal/contracts/oracle_contract/go.mod]: Fixed.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**